config/swagger.yaml

```diff
@@ -2459,36 +2903,72 @@ definitions:
     type: array
     items:
       $ref: '#/definitions/Tx'
+  RegisteredOracle:
+    type: object
+    properties:
+      id:
```

Comment:
I see property `address` changed to `id`, though this impacts only response value of `/oracles` internal obsolete path. So ok, and I think this in itself is not a reason for release notes line.